sql/updates/world/master/2016_11_xx_00_world.sql

```diff
@@ -0,0 +1,335 @@
+-- Arch Druid Hamuul Runetotem Texts
+DELETE FROM `creature_text` WHERE `entry`=53876;
+INSERT INTO `creature_text` (`entry`, `groupid`, `id`, `text`, `type`, `language`, `probability`, `emote`, `duration`, `sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(53876, 0, 0, 'Yes Cenarius, we must maintain a constant vigil over this realm.  But let us celebrate this day and the great victory we have earned here.', 14, 0, 100, 0, 0, 25168, 52575, 1, 'Arch Druid Hamuul Runetotem - Ragnaros Heroic Outro');
+-- Lava Scion
+DELETE FROM `creature_text` WHERE `entry`=53231;
+INSERT INTO `creature_text` (`entry`, `groupid`, `id`, `text`, `type`, `language`, `probability`, `emote`, `duration`, `sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(53231, 0, 0, '|TInterface\\Icons\\inv_elemental_mote_fire01.blp:16|tYou are about to burst into |cFFFF0000|Hspell:100460|h[Blazing Heat]|h|r!', 42, 0, 100, 0, 0, 0, 52819, 0, 'Lava Scion - Blazing Heat Warning');
+
+-- Achievement - Ragnar-O's
+DELETE FROM `criteria_data` WHERE `criteria_id`=17863;
+INSERT INTO `criteria_data` (`criteria_id`, `type`, `ScriptName`) VALUES (17863, 11, 'achievement_ragnar_os');
+
+-- Add Deluge spell click to Cloudburst
+DELETE FROM `npc_spellclick_spells` WHERE `npc_entry`=54147;
+INSERT INTO `npc_spellclick_spells` (`npc_entry`, `spell_id`, `cast_flags`, `user_type`) VALUES (54147, 100713, 3, 1);
+
+-- Ragnaros Kill Credit (DND)
+DELETE FROM `spelleffect_dbc` WHERE `Id`=155917;
+INSERT INTO `spelleffect_dbc` (`Id`, `Effect`, `EffectRadiusIndex`, `EffectImplicitTargetA`, `EffectImplicitTargetB`, `EffectSpellId`, `EffectIndex`) VALUES (155917, 3, 28, 22, 7, 102237, 0);
+
+-- Molten Elemental - Molten Seed Scale Aura
+DELETE FROM `spelleffect_dbc` WHERE `Id`=155918;
+INSERT INTO `spelleffect_dbc` (`Id`, `Effect`, `EffectApplyAuraName`, `EffectBasePoints`, `EffectDamageMultiplier`, `EffectImplicitTargetA`, `EffectSpellId`, `EffectIndex`) VALUES (155918, 6, 61, -20, 1, 1, 100170, 0);
+
+-- Ragnaros Kill Credit (DND)
+DELETE FROM `spell_dbc` WHERE `Id`=102237;
+INSERT INTO `spell_dbc` (`Id`, `Attributes`, `AttributesEx`, `AttributesEx2`, `AttributesEx3`, `AttributesEx6`, `Comment`) VALUES (102237, 536871296, 1056, 5, 268894464, 8192, 'Ragnaros Kill Credit (DND)');
+
+-- Molten Elemental - Molten Seed Scale Aura
+DELETE FROM `spell_dbc` WHERE `Id`=100170;
+INSERT INTO `spell_dbc` (`Id`, `Attributes`, `AttributesEx`, `DurationIndex`, `SpellAuraOptionsId`, `Comment`) VALUES (100170, 256, 1024, 21, 702, 'Molten Elemental - Molten Seed Scale Aura');
+
+-- Burning Wounds 5 second cooldown
+DELETE FROM `spell_proc_event` WHERE `entry`=99399;
+INSERT INTO `spell_proc_event` (`entry`, `SchoolMask`, `SpellFamilyName`, `SpellFamilyMask0`, `SpellFamilyMask1`, `SpellFamilyMask2`, `spellFamilyMask3`, `procFlags`, `procEx`, `ppmRate`, `CustomChance`, `Cooldown`) VALUES (99399, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5);
+
+DELETE FROM `spell_script_names` WHERE `spell_id` IN
```

Comment:
Ideally we should delete by `ScriptName` and not by `spell_id` (or not by `spell_id` alone). I don't remember the exact rationale behind it, but I presume it is because the same `ScriptName` could in theory be stored with several `spell_id`s.